.gitlab-ci.yml

```diff
@@ -0,0 +1,34 @@
+stages:
+  - build
+  - push
+
+build site:
```

Comment:
having the build as a separate "job" here means that it's possible that a different runner could get the push job than the build and that would not work at all I think. I believe for this you should just move this script to the push jobs:

```yaml
push dev:
  tags:
    - meao
  stage: push
  only: dev
  script:
    - bin/build.sh dev
    - bin/push-gh-pages.sh
```